src/components/Navigation.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeft from '@material-ui/icons/ChevronLeft';
+import MoreHoriz from '@material-ui/icons/MoreHoriz';
+
+const styles = {
+  root: {
+    flexGrow: 1,
+    boxShadow: 'none'
+  },
+  grow: {
+    flexGrow: 1,
+    textDecoration: 'none'
+  },
+  menuButton: {
+    marginLeft: -12,
+    marginRight: 20
+  },
+};
+
+function Navigation({
```

Comment:
This component is missing badges for indicating a number and image as shown for the navigation components labeled 'Chat' in Figma.